src/scripts/main.js

```diff
@@ -1,3 +1,24 @@
 'use strict';
 
-// write code here
+const tree = document.querySelector('.tree');
+
+for (const li of tree.querySelectorAll('li')) {
```

Comment:
```suggestion
const list = tree.querySelectorAll('li');

for (const li of list) {
```
Так було б наглядніше